sdk/transaction-builder/src/errmap.rs

```diff
@@ -0,0 +1 @@
+../../../language/move-prover/errmapgen/src/errmap.rs
```

Comment:
Thoughts on making this a separate crate and publishing it to crates.io?

Reply:
Ideally, this is all a short term hack until we can publish the whole diem-framework-release to crates.io. It's probably not worth the effort to publish such a small crate, but I don't have a particularly strong opinion here.

Reply:
Having too many small crates is a rather large maintenance burden so I'm hesitant to want to add a bunch more small crates. That's why I suggested rolling these things into the existing crate